packages/ariakit/src/checkbox/__examples__/checkbox-controlled/test.tsx

```diff
@@ -0,0 +1,22 @@
+import { click, getByRole, render } from "ariakit-test-utils";
+import Example from ".";
+
+test("render checkbox", async () => {
+  render(<Example />);
+  expect(getByRole("checkbox")).toMatchInlineSnapshot(`
```

Comment:
I guess we should use `container` here.
```suggestion
  const { container } = render(<Example />);
  expect(container).toMatchInlineSnapshot(`
```